src/api/useDarkThemeMode.tsx

```diff
@@ -0,0 +1,11 @@
+import { useState } from "react";
+
+export const useDarkThemeMode = () => {
+	const [isDarkTheme, setIsDarkTheme] = useState(false);
+
+	const toggleTheme = () => {
+		setIsDarkTheme((prevState) => !prevState);
+	};
+
+	return { isDarkTheme, setIsDarkTheme, toggleTheme };
```

Comment:
maybe some memoisation here would be useful?

Reply:
after your information. not yet with memo :)